docs/infrastructure/self-hosted/self-hosted-requirements.md

```diff
@@ -217,9 +219,52 @@ The following is an example architecture for the installation of the Mia-Platfor
+* (Optional) Velero for Disaster Recovery
+
+:::note
+These are minimum requirements for your cluster. These requirements have a scale factor that must be taken in account when estimating your cluster size.
+:::
+
+| **Tool**                     | **Namespace CPU limits** | **Namespace RAM limits** | **Scales with**                                               |
+|------------------------------|--------------------------|--------------------------|---------------------------------------------------------------|
+| Monitoring stack             | 1 core                   | 3 GiB                    | Number of cluster nodes, number of metrics                    |
+| Logging stack                | 0.5 core                 | 1 GiB                    | Number of cluster nodes, number of applications logs, traffic |
+| Traefik Ingress Controller   | 0.3 core                 | 300 MiB                  | Traffic                                                       |
+| Velero for Disaster Recovery | 0.2 core                 | 400 MiB                  | -                                                             |
+
+In total, the minimum requirements in order to host all the above tools in the Kubernetes cluster are about 2 cores and 5 GiB of memory. Above values will scale with the number of nodes, traffic, applications.
+
+Velero, in addition, needs a S3-like bucket storage for keeping the Kubernetes cluster backup.
+
+Grafana Mimir will be installed in the same Kubernetes cluster hosting Mia-Platform Console. The minimum requirements needed for Grafana Mimir are 2 cores and 4 GiB of memory.
+
+For a simple Mia-Platform Console Project with 3 environments, a minimum of 6 cores and 12 GiB of memory are needed. Of course this is just an indicative estimation and the right sizing depends on your application.
+
+### Suggested resource allocation for your runtime
+
+:::note
+The following requirements are just an approximate reference. For a more accurate estimate, an assessment of your application and architecture is needed.
+:::
+
+:::info
+To size your cluster you should reserve a minimum of 3 cores for Operations tools and minimum 2 cores for each Runtime Environment managed by Mia-Platform. A Project can managed one or more Runtime Environment.
+:::
+
+Cluster sizing examples:
 
-The Console Super User role has been designed exclusively for backoffice administration purposes. 
-When assigned to a user, this role implies full visibility and management of all CMS resources.
+* 1 Mia-Platform Project with 3 Runtime Environments: 5 worker nodes (2 cores, 4 GiB RAM each). 4 cores for Operations tools and 6 cores for your applications
+* 3 Mia-Platform Project with 3 Runtime Environments: 12 worker nodes (2 cores, 4 GiB RAM each). 5 cores for Operations tools and 19 cores for your applications
+* 5 Mia-Platform Project with 3 Runtime Environments: 18 worker nodes (2 cores, 4 GiB RAM each). 6 cores for Operations tools and 30 cores for your applications
```

Comment:
The minimum license is of 20 cores; based on this assumption, can be these examples misleading for the user?
Moreover, I propose to add a warning about the scalability of your services, e.g. "You should always have some free resources in case you need to scale your runtime based on traffic peaks or similar cases".

Reply:
I could remove the 268 line to avoid the mismatch with the license.

Reply:
Great